dali/test/python/test_dali_tf_dataset_eager.py

```diff
@@ -35,7 +37,10 @@ def test_tf_dataset_cpu():
     run_tf_dataset_eager_mode('cpu')
 
 # Return differently sized images to check if DALIDataset can handle this case gracefully
-@raises(tf.errors.FailedPreconditionError)
+@raises(tf.errors.FailedPreconditionError, glob='*Batch output at index * from DALI pipeline is not uniform - individual '
```

Comment:
You can remove leading and trailing `*` from glob pattern. They are not needed, as glob to regexp converted we use adds them automatically.

Reply:
You can skip most of the message and only match relevant fragments, making some slack for future adjustment of the error messages.

Reply:
I removed all the leading and trailing * and shortened the patterns to the essential parts.